Assets/SEE/Controls/Actions/ShowEdges.cs

```diff
@@ -29,6 +34,23 @@ public class ShowEdges : InteractableObjectAction
         /// </summary>
         private AbstractSEECity codeCity;
 
+        /// <summary>
+        /// A token that's used to cancel the transitive edge toggling.
+        /// We need to use this instead of the edge operator's built-in conflict mechanism
+        /// because the edge operator only controls a single edge, instead of the whole transitive closure.
+        /// </summary>
+        private CancellationTokenSource edgeToggleToken;
+
+        /// <summary>
+        /// The toggle that is used to determine whether an edge is currently selected.
+        /// </summary>
+        private const string EdgeIsSelectedToggle = "IsSelected";
+
+        /// <summary>
+        /// The delay between each depth level when showing/hiding the transitive closure of edges.
```

Comment:
The unit is second, I suppose? The comment should mention that.

Reply:
I don't think mentioning a unit here makes much sense, given that the type of this field is `TimeSpan` (which has units built into it).

Reply:
Okay, I see. Thanks for the clarification.